README.md

```diff
@@ -328,6 +328,16 @@ More information about the pack can be found here [Github Student Developer Pack
 - [Resources 5](https://join.codecademy.com/learn/introduction-to-javascript/)
 - [Resources 6](https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/)
 
+## Mini Projects using HTML,CSS and Javascript
```

Comment:
Change title to `tutorials of creating mini projects using html , css , js ` @Uma-95

Reply:
@epicadidash Done.